### PR TITLE
chore: bump sd-boot to v254-rc1

### PR DESCRIPTION
**Pull request by frezbo**
Bump sd-boot.
Fix parsing PE executable offsets.
Set the PE file alignment to be 512 bytes.

### CODE REVIEW

File: cmd/installer/pkg/iso.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	// UKIISOSize is the size of the UKI ISO.
-	UKIISOSize = 120 * 1024 * 1024
+	UKIISOSize = 80 * 1024 * 1024
```
Comment (frezbo) [inline cmd/installer/pkg/iso.go:22]: fixing the alignment means we can use a proper ISO size.

File: hack/ukify/main.go [hunk 2]
```diff
@@ -107,9 +107,16 @@ func buildUKI(source, output string, sections []section) error {
 	// find the first VMA address
 	lastSection := peFile.Sections[len(peFile.Sections)-1]
 
-	const alignment = 0xfff
+	// align the VMA to 512 bytes
+	// https://github.com/saferwall/pe/blob/main/helper.go#L22-L26
+	const alignment = 0x200
```
Comment (smira) [inline hack/ukify/main.go:112]: this should be `0x1ff` (alignment - 1) as we use it this way

**Review by smira — APPROVED** [review]

**Pull request merged**